src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -141,12 +143,27 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
           </div>
         </DialogContent>
         <DialogActions>
-          <Button color="secondary" variant="outlined" form="activate-work-package-form" onClick={onHide}>
-            Cancel
-          </Button>
-          <Button color="success" variant="contained" type="submit" form="activate-work-package-form">
-            Submit
-          </Button>
+          <Box textAlign="left" sx={{ my: 2 }}>
+            <NERFailButton
+              color="secondary"
+              variant="outlined"
+              form="activate-work-package-form"
+              onClick={onHide}
+              sx={{ mx: 1 }}
```

Comment:
make the margins a tiny bit bigger, Reid wanted about 2px between the buttons (idk where these numbers are coming from but its for the greater good)

Reply:
Just pushed the changes you requested to the buttons, lmk if anything else needs fixing:

<img width="212" alt="image" src="https://user-images.githubusercontent.com/55080943/216150140-228d5829-633f-4cd9-9573-eaa86f1647b3.png">